Readme.md

```diff
@@ -1,81 +1,97 @@
-Paramcoq
-========
+# Paramcoq
 
-[![Build Status](https://travis-ci.com/coq-community/paramcoq.svg?branch=master)](https://travis-ci.com/coq-community/paramcoq/branches)
+[![CI][action-shield]][action-link]
+[![Contributing][contributing-shield]][contributing-link]
+[![Code of Conduct][conduct-shield]][conduct-link]
+[![Zulip][zulip-shield]][zulip-link]
 
-The plugin is still in an experimental state. 
-It is not very user friendly (lack of good error messages) and still contains bugs. 
-But is useable enough to "translate" a large chunk of standard library. 
+[action-shield]: https://github.com/coq-community/paramcoq/workflows/CI/badge.svg?branch=master
+[action-link]: https://github.com/coq-community/paramcoq/actions?query=workflow%3ACI
 
-Compilation 
------------
+[contributing-shield]: https://img.shields.io/badge/contributions-welcome-%23f7931e.svg
+[contributing-link]: https://github.com/coq-community/manifesto/blob/master/CONTRIBUTING.md
 
-The plugin is believed to work with Coq ≥ 8.7.
-One way to test the plugin is to follow the following steps:
+[conduct-shield]: https://img.shields.io/badge/%E2%9D%A4-code%20of%20conduct-%23f15a24.svg
+[conduct-link]: https://github.com/coq-community/manifesto/blob/master/CODE_OF_CONDUCT.md
 
-* Retrieve the plugin and compile it:
+[zulip-shield]: https://img.shields.io/badge/chat-on%20zulip-%23c1272d.svg
+[zulip-link]: https://coq.zulipchat.com/#narrow/stream/237663-coq-community-devs.20.26.20users
 
-        git clone ...
-        cd paramcoq
-        git checkout master  # or v8.x depending on Coq version
-        make
-        make install
 
-* To test the plugin:
 
-        cd test-suite
-        make ide
+The plugin is still in an experimental state. It is not very user friendly (lack of good error messages) and still contains bugs. But is useable enough to "translate" a large chunk of standard library.
+
+
+## Meta
+
+- Author(s):
+  - Chantal Keller (initial)
+  - Marc Lasson (initial)
+  - Abhishek Anand
+  - Pierre Roux
+  - Emilio Jesús Gallego Arias
+  - Cyril Cohen
+  - Matthieu Sozeau
+- Coq-community maintainer(s):
+  - Pierre Roux ([**@proux01**](https://github.com/proux01))
+- License: [MIT](LICENSE)
+- Compatible Coq versions: The master branch tracks the development version of Coq, see releases for compatibility with released versions of Coq.
+
+- Additional dependencies: none
+- Coq namespace: `Param`
+- Related publication(s): none
```

Comment:
Could you add https://hal.archives-ouvertes.fr/hal-00730913/ while you are at it?